test/better_words/words_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule BetterWords.WordsTest do
```

Comment:
This file is 100% generated. I write my tests differently, but these are good tests.